src/components/LemonReset/LemonReset.js

```diff
@@ -2,7 +2,7 @@
 
 import * as React from 'react';
 import classNames from 'classnames';
-import styles from './LemonReset.scss';
+import styles from '../../../lib/components/LemonReset/LemonReset.css';
```

Comment:
This is admittedly janky, but eslint will complain if I import from `./LemonReset.css` (because the file doesn't exist in `./src`).

If people prefer, we could do it like this instead:

```
// eslint-disable-next-line import/no-unresolved
import styles from './LemonReset.css';
```